### DISCUSSION TITLE
doc: Postgres Partitioned Table Support

### DISCUSSION THREAD

**Discussion by lcmgh** [SeaQL/sea-orm#1951] [General]
I have not found the keyword partitions on https://www.sea-ql.org/SeaORM/ but found it was mentioned in release notes: https://github.com/SeaQL/sea-orm/releases/tag/0.12.1

**Comment by gitmalong** [comment]
There is no way to created partitioned tables with https://docs.rs/sea-orm-migration/0.12.4/sea_orm_migration/prelude/struct.TableCreateStatement.html

**Comment by tyt2y3** [comment]
I guess in that workflow, the partitioned table could be created in some other means, e.g. raw SQL or it's an existing database.

**Comment by gitmalong** [comment]
I'd love to see it as option in SeaORM when creating tables.

**Reply by tyt2y3** [reply]
Sure
